public/style.css

```diff
@@ -0,0 +1,106 @@
+.app_name {
+  font-family: "Motiva Sans", Sans-serif;
+  font-weight: normal;
+  padding-left: 0px;
+  color: white;
+  font-size: 26px;
+  line-height: 32px;
+  text-overflow: ellipsis;
+  margin-left: 157px;
+  margin-top: 30px;
+}
+
+.background {
+  background-color: #1b2838;
+}
+
+br {
+  content: "";
+  margin: 3em;
+  display: block;
+  font-size: 24%;
+}
```

Comment:
This seems odd. What's the intent with the `<br />` elements here? I think a different tactic might be better for adding what I'm assuming is vertical spacing in your presentation?